### ISSUE TITLE
Custom Row Style Field Hook

### ISSUE THREAD

**Issue by markifornia** [siteorigin/docs#67]
The code here - https://siteorigin.com/docs/page-builder/hooks/custom-row-settings/

```
function custom_row_style_attributes( $attributes, $args ) {
    if( !empty( $args['parallax'] ) ) {
        array_push($attributes['class'], 'parallax');
    }

    return $attributes;
}

add_filter('siteorigin_panels_row_style_attributes', 'custom_row_style_attributes', 10, 2);
```

"To add a new attribute, add both a new key and value to $attributes. Example: $attributes['data-video-id'] = $args['video-id']; will add data-video-id="9" to the row element."

Thanks for building these hooks but this example isn't clear. I am looking to add a background image using data-bg. So my code would look like this:

```
function custom_row_style_fields($fields) {
  $fields['lazyload'] = array(
    'name'        => __('Lazy Load', 'siteorigin-panels'),
    'type'        => 'text',
    'group'       => 'attributes',
    'description' => __('Add image url for lazy loading', 'siteorigin-panels'),
    'priority'    => 8,
  );

  return $fields;
}

add_filter( 'siteorigin_panels_row_style_fields', 'custom_row_style_fields' );

function custom_row_style_attributes( $attributes, $args ) {
    if( !empty( $args['lazyload'] ) ) {
        array_push($attributes['data-bg'], $args['lazyload']);
    }

    return $attributes;
}

add_filter('siteorigin_panels_row_style_attributes', 'custom_row_style_attributes', 10, 2);

```

However, only 'data-bg' is being added to the element but I am unable to capture the value. 

Appreciate your help.

Thanks,
Mark

**Comment by AlexGStapleton** [comment]
The provided example code for `siteorigin_panels_row_style_attributes` is set up to work for the previously provided example code for `siteorigin_panels_row_style_fields`.

You'll need to adjust your code if you're looking to output an attribute as `array_push` cannot be used to do what your after. The following text explains how to adjust your code:

> To add a new attribute, add both a new key and value to $attributes. Example: **$attributes['data-video-id'] = $args['video-id'];** will add data-video-id="9" to the row element.

The bolded part of the above quote is what you need to do.
Basically, for your code to work as desired, you need to replace:

`array_push($attributes['data-bg'], $args['lazyload']);
`
With: 

`$attributes['data-bg'] = $args['lazyload'];`

**Comment by markifornia** [comment]
@AlexGStapleton thank you this is what I was looking for - I was able to insert the key value pair into 
the target element. 

Great plugin, thank you for providing these hooks!

Cheers!

**Comment by AlexGStapleton** [comment]
Great to hear that helped! :)
Thank you for your kind words.
